data/food/export_agb/export_builder.py

```diff
@@ -125,38 +153,42 @@ def compute_new_processes(activities, ingredients_base):
                 processes_to_add.append({"name": variant["simple_ingredient_default"]})
                 processes_to_add.append({"name": variant["simple_ingredient_variant"]})
 
-    print(
-        f"{len(processes_to_add)} procédés construits provenant de {INGREDIENTS_BASE}"
-    )
+    print(f"{len(processes_to_add)} ingrédients construits depuis {INGREDIENTS_BASE}")
 
     processes_to_export += processes_to_add
     print(
-        f"Total de {len(processes_to_export)} procédés à exporter, sélectionnés depuis {PROCESSES2EXPORT}"
+        f"{len(processes_to_export)} procédés exportés depuis depuis {PROCESSES2EXPORT}"
     )
 
     activities = {}
-    for p in processes_to_export:
-        results = db.search(p["name"])
-        assert len(results) >= 1, f"Research doesn't give a result: {p['name']}"
+    for i, process in enumerate(processes_to_export):
+        results = db.search(process["name"])
+        assert (
+            len(results) >= 1
+        ), f"In {PROCESSES2EXPORT}:{i}, searching this \"name\" field doesn't give a result: {process['name']}"
         activity = results[0]
-        activities[p["name"]] = {
+        activities[activity["Process identifier"]] = {
             "activity": activity,
-            "export_data": p,
+            "export_data": process,
```

Comment:
Now we're using "search strings" as process identifier proxies, it may be worth renaming that specific column and/or add a specific comment about this.

Reply:
I suppose you mean the `name` column of the csv file. I renamed to `search`